ext/symengine/symengine_utils.c

```diff
@@ -59,10 +61,16 @@ void sympify(VALUE operand2, basic_struct *cbasic_operand2) {
             break;
 
         case T_DATA:
+            c = rb_obj_classname(operand2);
+            if(strcmp(c, "BigDecimal") == 0){
+                c = RSTRING_PTR( rb_funcall(operand2, rb_intern("to_s"), 1, rb_str_new2("F")) );
+                real_mpfr_set_str(cbasic_operand2, c, 200);
```

Comment:
SymEngine might not have been compiled with MPFR. So you cannot expect `real_mpfr_set_str` to be available all the time.